pages/acknowledgements.js

```diff
@@ -2,14 +2,71 @@ import React from "react";
 
 import SharedLayout from "components/SharedLayout";
 import BackButton from "components/navigation/BackButton";
-import PlaceholderText from "components/mock/Placeholder";
 
 export default function Acknowledgements() {
     return (
         <SharedLayout>
             <BackButton className="mt-3 mb-2" />
             <h1>Acknowledgements</h1>
-            <PlaceholderText className="mt-5 mb-2" />
+            <p>We want to take a moment to acknowledge that the Policy Map and Scorecard could not have been made possible without the support and contributions made by the law firms, research volunteers and all those involved in this project! We thank you for all of your work and contribution to making this tool a success!</p>
+            <h4>Pro Bono Partners</h4>

```

Comment:
Another heading rule:
>A common navigation technique for users of screen reading software is jumping from heading to heading to quickly determine the content of the page. Because of this, it is important to not skip one or more heading levels. Doing so may create confusion, as the person navigating this way may be left wondering where the missing heading is.
>
>Don't
> ```
> <h1>Heading level 1</h1>
> <h3>Heading level 3</h3>
> <h4>Heading level 4</h4>
> ```
> Do
> ```
> <h1>Heading level 1</h1>
> <h2>Heading level 2</h2>
> <h3>Heading level 3</h3>
> ```

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes

Once again, if time allows, it would be nice to change the h4 elements to h3 elements and style them. If not, we can maybe write up some issues and deal with it later.
```suggestion
            <h3>Pro Bono Partners</h3>
```